shell/browser/native_browser_view_mac.mm

```diff
@@ -232,10 +232,10 @@ - (BOOL)mouseDownCanMoveWindow {
 void NativeBrowserViewMac::UpdateDraggableRegions(
     const std::vector<gfx::Rect>& drag_exclude_rects) {
   auto* iwc_view = GetInspectableWebContentsView();
-  auto* web_contents = GetWebContents();
+  auto* web_contents = inspectable_web_contents_->GetWebContents();
```

Comment:
since we know that there's a possibility `inspectable_web_contents_` is `nullptr`, we need to check that it's not before calling `GetWebContents()`

Reply:
@mlaurencin -

> `auto* web_contents = inspectable_web_contents_->GetWebContents();`

as it's written right now makes it such that if `inspectable_web_contents_` is `nullptr`, there will still be a crash. This is because calling a function on a `nullptr` is undefined behavior, but in this case an access violation as it would be trying to access memory which is no longer allocated. We need to first check `> `inspectable_web_contents_ != nullptr` _before_ trying to call a member function on it.